.github/workflows/e2e.yml

```diff
@@ -0,0 +1,67 @@
+name: End to End Tests
+on: 
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'develop'
+        type: string
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest]
+        vscodeVersion:
+          - stable
+    runs-on: ${{ matrix.os }}
+    steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+          with:
+            path: ./salesforcedx-vscode
+            ref: ${{ github.event.ref }}
+        - uses: actions/setup-node@v3
+          with:
+            node-version-file: 'salesforcedx-vscode/.nvmrc' 
+            cache: npm
+            cache-dependency-path: |
+              salesforcedx-vscode/package-lock.json
+              salesforcedx-vscode-automation-tests/package-lock.json
+        - name: Clone automation tests
+          uses: actions/checkout@v2
+          with:
+            repository: forcedotcom/salesforcedx-vscode-automation-tests
+            path: salesforcedx-vscode-automation-tests
+            ref: ${{ inputs.automationBranch }}
+        - name: Install Extension Dependencies
+          run: |
+            npm install
+          working-directory: salesforcedx-vscode
+        - name: Install Test Dependencies
+          run: |
+            npm install
+          working-directory: salesforcedx-vscode-automation-tests
+        - name: Install the SFDX CLI
+          run: npm install -g sfdx-cli
+        - name: Verify CLI
+          shell: bash
+          run: |
+            set -e
+            sfdx version
+            SFDX_CLI_VERSION=$(sfdx version)
+            if [[ ((`echo $SFDX_CLI_VERSION | grep -c "sfdx-cli/"` > 0))]]
```

Comment:
I'm not sure line 54 will work on Windows

Reply:
Good catch - this one I grabbed from the CLI, but they were using for Docker builds specifically, which was running on ubuntu. For now it's fine if we're only running against mac, but we could take the time to change it now, or kick the can to fix this once we start working on Windows. I also have a vague memory of this working when we did the initial windows tests, but re-verification/testing would definitely be necessary to confirm.

Reply:
Also it looks like this does work on Windows as well 😎  https://github.com/randi274/salesforcedx-vscode-fork/actions/runs/4788500534/jobs/8515192099